lib/index.js

```diff
@@ -120,8 +120,13 @@ if (!realmConstructor) {
 require('./extensions')(realmConstructor);
 
 if (realmConstructor.Sync) {
-  require('./notifier')(realmConstructor);
-  Object.defineProperty(realmConstructor, 'Worker', {value: require('./worker')});
+  switch(getContext()) {
+    case 'nodejs':
+    case 'electron':
```

Comment:
Electron doesn’t have the global notifier, so we shouldn’t apply this for it.

Reply:
:+1: